notebooks/1.2-pyc-eda-mix.py

```diff
@@ -0,0 +1,500 @@
+# - Alley
+# - Neighborhood
+# - Condition 1
+# - Condition 2
+
+# ## MSZoning
+# 
+# Identifies the general zoning classification of the sale.
+# 		
+#        A	Agriculture
+#        C	Commercial
+#        FV	Floating Village Residential
+#        I	Industrial
+#        RH	Residential High Density
+#        RL	Residential Low Density
+#        RP	Residential Low Density Park 
+#        RM	Residential Medium Density
+
+multiple_ecdf('MSZoning')
+
+
+# **Insights**
+# 
+# - The RH category can be joined with the RM one
+# - C category have few data, but this graphic doesn't give us an option to join with another column
+
+# ## Street
+# 
+# 
+# Type of road access to property
+# 
+#        Grvl	Gravel	
+#        Pave	Paved
+
+train['Street'].value_counts()
+
+
+# **Insights**
+# 
+# - Few values aren't equal to Pave, so maybe we should just drop this column
+
+# ## Alley
+# 
+# Type of alley access to property
+# 
+#        Grvl	Gravel
+#        Pave	Paved
+#        NA 	No alley access
+
+train['Alley'].value_counts()
+
+
+multiple_ecdf('Alley')
+
+
+# **Insights**
+# 
+# - Almost all values are Nan
+# - The not nan variables appears to be very important
+# - I think that we should just fill the nan values in this column
+
+# ## Neighborhood
+# 
+# Physical locations within Ames city limits
+# 
+#        Blmngtn	Bloomington Heights
+#        Blueste	Bluestem
+#        BrDale	Briardale
+#        BrkSide	Brookside
+#        ClearCr	Clear Creek
+#        CollgCr	College Creek
+#        Crawfor	Crawford
+#        Edwards	Edwards
+#        Gilbert	Gilbert
+#        IDOTRR	Iowa DOT and Rail Road
+#        MeadowV	Meadow Village
+#        Mitchel	Mitchell
+#        Names	North Ames
+#        NoRidge	Northridge
+#        NPkVill	Northpark Villa
+#        NridgHt	Northridge Heights
+#        NWAmes	Northwest Ames
+#        OldTown	Old Town
+#        SWISU	South & West of Iowa State University
+#        Sawyer	Sawyer
+#        SawyerW	Sawyer West
+#        Somerst	Somerset
+#        StoneBr	Stone Brook
+#        Timber	Timberland
+#        Veenker	Veenker
+# 			
+
+multiple_ecdf('Neighborhood')
+
+
+# **Insights**
+# 
+# - A LOT of categories
+# - It's hard to group the categories, so we can use an unsupervised method to group then
+
+# ## Condition 1 and 2
+# 
+# Proximity to various conditions
+# 	
+#        Artery	Adjacent to arterial street
+#        Feedr	Adjacent to feeder street	
+#        Norm	Normal	
+#        RRNn	Within 200' of North-South Railroad
+#        RRAn	Adjacent to North-South Railroad
+#        PosN	Near positive off-site feature--park, greenbelt, etc.
+#        PosA	Adjacent to postive off-site feature
+#        RRNe	Within 200' of East-West Railroad
+#        RRAe	Adjacent to East-West Railroad
+# 	
+
+multiple_ecdf('Condition1')
+
+
+multiple_ecdf('Condition2')
+
+
+# **Insights**
+# 
+# - almost all are 'Norm"
+# - We can take just the more often categories, like "Feeddr" and "Artery" and make bool columns with these ones
```

Comment:
So should join the other categories in just one ?
So in this way we would have 3 categories: 'Feeddr', 'Artery' and 'other'